packages/cosmic-swingset/lib/ag-solo/vats/lib-wallet.js

```diff
@@ -366,7 +366,15 @@ export async function makeWallet(
         return E(cancelObj).cancel();
       });
       idToOfferHandle.set(id, offerHandle);
-      subscribeToNotifier(id, offerHandle);
+      // The offer might have been added to the book, or it might have been
```

Comment:
```suggestion
      // The offer might have been postponed, or it might have been
```

I prefer not to tie this implementation to the idea of an "order book".  There might be other reasons to postpone the offer.

Reply:
done